rpc.go

```diff
@@ -1,7 +1,28 @@
 package rpc
 
-func Ping(params interface{}, id ...interface{}) string {
+import (
+	"encoding/json"
+)
+
+// RPC struct
+type RPC struct {
+	base
+}
+
+// NewRPC is a constructor of RPC
+func NewRPC(node string) RPC {
+	return RPC{base: newBase(node, nil)}
+}
+
+// Ping sends request to node
+func (rpc RPC) Ping(params interface{}, id interface{}) {
```

Comment:
Since JavaScript supports default value in function arguments, it was fine to get `params` and `id` argument in all RPC functions. However, in Golang, the default parameter does not exist. How about removing params and using a random value as id?